src/config/types.ts

```diff
@@ -0,0 +1,5 @@
+import { IMessage } from 'react-native-gifted-chat';
+
+export interface ChatMessage extends Omit<IMessage, '_id'> {
```

Comment:
It's confusing, is that supposed to be our own `ChatMessage` definition then? But it seems to be from the library you are using... Consider renaming to `GiftedChatMessage`